dsc/decisionMaker.md

```diff
@@ -0,0 +1,36 @@
+# Desired State Configuration Overview for Decision Makers #
+
+This document describes the business benefits of using PowerShell Desired State Configuration (DSC). It is not a technical guide.
+
+## What Is Desired State Configuration? ##
+
+Windows PowerShell Desired State Configuration (DSC) provides a configuration platform built into Windows that is based on open standards. DSC is flexible enough to function reliably and consistently in each stage of the deployment lifecycle (development, test, pre-production, production), as well as during scale-out. 
+
+DSC centers around the idea of "[configurations](https://msdn.microsoft.com/en-us/powershell/dsc/configurations)", which are easy-to-read documents that describe an environment made up of computers ("nodes") with specific characteristics. These characteristics can be as simple as ensuring a specific Windows feature is enabled, or as complex as deploying SharePoint. 
+
+DSC also has monitoring and reporting built in. If a system is no longer compliant, DSC can raise an alert and act to correct the system. 
+
+## Benefits of Using Desired State Configuration ##
+
+Configurations are designed to be easily read, stored, and updated. Configurations simply declare the state target devices should be in, instead of writing instructions for how to put them in that state. This makes it much less costly for to learn, adopt, implement, and maintain configuration through DSC. 
```

Comment:
Tiny typo in bold:

"This makes it much less costly **for** to learn, adopt, implement, and maintain configuration through DSC."